lib/actions/model.js

```diff
@@ -108,4 +108,11 @@ module.exports = {
       return validationErrors;
     };
   },
+
+  set(model, ...args) {
+    return () => {
+      model.set(...args);
```

Comment:
The issue we have is there is no ID. This came about specifically from dealing with an unpersisted model that's shared between a React component and Marionette code.

Maybe this isn't that useful to move to `brainstem-redux`? I think we need to sync up with @naganowl on this.